CHANGELOG.md

```diff
@@ -6,6 +6,8 @@ and this project adheres to [Semantic Versioning](http://semver.org/spec/v2.0.0.
 
 ## [Unreleased]
 
+## [3.1.0] - 2020-10-08
+
 ### Added
 - Module now fully supports the use of `cert_file` parameter, which accepts the path to the PEM-encoded
```

Comment:
Very minor point... try to avoid using the word "supports" when describing new features.  Can be simpler/more straightforward to just say what it does.

Reply:
Yeah I know you're not a fan of it so I've been avoiding its use (😄) but sometimes it just fits better in the text. In this case it was hard to explain we went from partial to full enablement without "supports".